presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -1517,4 +1523,9 @@ public static boolean isCheckAccessControlOnUtilizedColumnsOnly(Session session)
     {
         return session.getSystemProperty(CHECK_ACCESS_CONTROL_ON_UTILIZED_COLUMNS_ONLY, Boolean.class);
     }
+
+    public List<PropertyMetadata<?>> getSessionProperties()
```

Comment:
Why did this get deleted/added. Maybe whitespace change or something. Just restore the original.